pkg/controller/importconfig/importconfig_controller.go

```diff
@@ -142,10 +119,95 @@ func (r *ReconcileImportConfig) Reconcile(ctx context.Context, request reconcile
 	return reconcile.Result{}, nil
 }
 
-func klusterletNamespace(managedCluster *clusterv1.ManagedCluster) string {
-	if klusterletNamespace, ok := managedCluster.Annotations[constants.KlusterletNamespaceAnnotation]; ok {
-		return klusterletNamespace
+func generateDefaultImportSecret(ctx context.Context, bootstrapKubeconfig, expiration []byte,
+	managedClusterName string, managedClusterAnnotations map[string]string,
+	clientHolder *helpers.ClientHolder) (*corev1.Secret, error) {
+	// generate manifests data in the format of bytes
+	yamlcontent, err := NewKlusterletManifestsConfig(
+		append(append([]string{}, KlusterletOperatorFiles...), KlusterletFiles...),
+		operatorv1.InstallModeDefault,
+		managedClusterName,
+		klusterletNamespace(managedClusterAnnotations),
+		bootstrapKubeconfig).
+		WithManagedClusterAnnotations(managedClusterAnnotations).
+		Generate(ctx, clientHolder)
+	if err != nil {
+		return nil, err
+	}
+
+	crdsV1beta1YAML, err := GenerateKlusterletCRDsV1Beta1()
```

Comment:
why we need two separated funcs for different version of crds?

Reply:
In the agent-registration case, we want to let user run one command to apply crd:

```
curl --cacert ca.crt -H "Authorization: Bearer $token" https://$agent_registration_host/agent-registration/crds/v1 | kubectl apply -f -
```

If we return 2 crds in one response, the `kubectl apply` may return error if the kubernetes doesn't support crds v1beta1.